CHANGELOG.md

```diff
@@ -11,6 +11,112 @@ All notable changes to Botonic will be documented in this file.
+  - Added `getAgentVacationRanges` to know availability of agents.
+  - Added more `router`tests.
+  - Type definitions improvement in `index.d.ts`.
+
+* [@botonic/react](https://www.npmjs.com/package/@botonic/react)
+
+  - General features:
+
+    - Type definitions improvement in `index.d.ts`.
+    - Improved `Text.serialize` method.
+
+  - New webchat features:
+
+    - **Breaking change**: Now the text of a `Button/Reply` is sent along with its payload. You can use `button: {messageType: 'postback'}` in `webchat/index.js` to enable the previous behavior.
+    - **Breaking change**: Markdown will be rendered by default within all the `Text` components. If you want to disable it, you can declare the component as follows:  
+      `<Text markdown={false}>Your text</Text>`.
+    - Be able to pass additional styles to `Custom Messages`.
+    - Persistent Menu customizable with property `customPersistentMenu` in `webchat/index.js`.
+    - Persistent Menu button customizable with property `customMenuButton`.
+    - Possibility to enable timestamps in messages.
+    - Added unit tests for Webchat.
+    - More adaptable Webchat in iOS.
+
+  - Multichannel:
+
+    - Customize `Multichannel` visualization.
+    - `Multichannel` to support letter indexes.
+    - Compact `Multichannel` `Text` and `Carousel`.
+    - `Multichannel` to allow separating messages with custom string.
+
+- [@botonic/plugin-contentful](https://www.npmjs.com/package/@botonic/plugin-contentful)
+  - `Comment` and `Chitchat` typing improvements.
+  - Allow passing `Normalizer` to `CMS`.
+  - Added Markup (WhatsApp Support) and Markdown parsing.
+  - Added stemmers/tokens for Russian.
+
+* [@botonic/plugin-inbenta](https://www.npmjs.com/package/@botonic/plugin-inbenta)
+
+  - Added new package `@botonic/plugin-inbenta` to integrate the Inbenta Knowledge Management API. Please refer to its README for more information.
+
+### Changed
+
+- [Project](https://github.com/hubtype/botonic)
+
+  - Move most `eslint` rules config to root `.eslintrc.js`.
+
+* [@botonic/react](https://www.npmjs.com/package/@botonic/react)
+
+  - Move `webchatReducer` to its own file.
+  - Split `webchatReducer` to reduce its complexity.
+  - Unify calls to `renderBrowser`/`renderNode`.
+  - Update `emoji-picker-react` to latest version `3.7.1` (it changes visually).
+
+- [@botonic/plugin-contentful](https://www.npmjs.com/package/@botonic/plugin-contentful)
+
+  - Upgrade `node-nlp` dependency.
+  - Use `ContextWithLocale` if locale required.
+
+### Fixed
+
+- [Project](https://github.com/hubtype/botonic)
+
+  - `pre-commit` was not aborting when `lint` failed
+  - Fix several `eslint` warnings.
+
+* [@botonic/cli](https://www.npmjs.com/package/@botonic/cli)
+  - Preserve `src/nlu` directories for all templates (added `.gitkeep` file). Remove them before running `botonic train`.
+  - Add missing calls with `await`.
+  - Use `rimraf` in favor of `fs.rmdirSync` which produced buggy behaviors.
```

Comment:
In release notes, I wouldn't explain how you solved the issues, but which problem you fixed